msi/release.sh

```diff
@@ -4,6 +4,7 @@ usage () {
 	cat <<EOF
 
 Usage: $0 [-o <dir>] <version> [<optional-package>...]
+   or: $0 --render-release-notes
```

Comment:
I am not really a fan of teaching each and every `release.sh` about `--render-release-notes`... I know, I did it myself, but that was only because I was lazy and I tried to avoid refactoring out the rendering code into a separate script.

In fact, I would rather see `render-release-notes.sh` to sprout a `--preview` option, and `installer/release.sh` losing its `--render-release-notes` option instead.